### PR TITLE
DGC-105 Responsive menu component

### PR DESCRIPTION
**Pull request by jonraedeke**
#105 

- Responsive
- Pure CSS
- Accessible
- Animated

### CODE REVIEW

File: docroot/themes/custom/twentyseventeen/sass/components/nav/_nav.scss [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      transition: opacity 0.5s ease;
+    }
+
+    a:hover svg {
+      opacity: 0.7;
+    }
+  }
+
+  &__logo-text {
+    @include element-invisible;
```
Comment (sarahjean) [inline docroot/themes/custom/twentyseventeen/sass/components/nav/_nav.scss]: Got an error here: `Error: no mixin named element-invisible`

**Comment by sarahjean** [comment]
@jonraedeke This needs some rebasing, and the one mixin issue, since I hit the compiling issue I wasn't able to look at the behavior. The twig doesn't look too bad to me.

**Comment by jonraedeke** [comment]
@sarahjean Thanks. I fixed the missing mixin issue.

**Comment by sarahjean** [comment]
Looks good.

**Pull request merged**